app-doc/zeal/zeal-0.7.0_p20231006.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+COMMIT=2cefcd45423e58283372a4c735609c284eded410
+inherit cmake xdg-utils
+
+DESCRIPTION="Offline documentation browser inspired by Dash"
+HOMEPAGE="https://zealdocs.org/"
+SRC_URI="https://github.com/zealdocs/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3"
```

Comment:
should be `GPL-3+`, see: https://github.com/zealdocs/zeal/blob/main/src/app/main.cpp#L9-L12

Reply:
Also added `CC0-1.0`, see https://github.com/zealdocs/zeal/blob/main/assets/freedesktop/org.zealdocs.zeal.appdata.xml.in#L5-L6